src/TitleBindPieces/TitleBindPieces.js

```diff
@@ -0,0 +1,121 @@
+import PropTypes from 'prop-types';
+import { useMemo } from 'react';
+import { FieldArray } from 'react-final-form-arrays';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  FormFooter,
+  handleKeyCommand,
+} from '@folio/stripes-acq-components';
+import {
+  checkScope,
+  HasCommand,
+  Pane,
+  Paneset,
+} from '@folio/stripes/components';
+import stripesFinalForm from '@folio/stripes/final-form';
+
+import { TitleBindPiecesList } from './TitleBindPiecesList';
+import { TitleBindPiecesCreateItemForm } from './TitleBindPiecesCreateItemForm';
+import { isRequiredFieldsFilled } from './utils';
+
+const FIELD_NAME = 'receivedItems';
+
+const TitleBindPieces = ({
+  form,
+  handleSubmit,
+  locations,
+  onCancel,
+  paneTitle,
+  pristine,
+  submitting,
+  values,
+  onSubmit,
+  isLoading,
+}) => {
+  const disabled = useMemo(() => {
+    if (!isRequiredFieldsFilled(values)) {
```

Comment:
Why don't you use `valid` value from the form state instead? If required fields are not populated, then `valid === false`